pymilo/streaming/pymilo_client.py

```diff
@@ -1,14 +1,24 @@
+from enum import Enum
 from .encryptor import DummyEncryptor
 from .compressor import DummyCompressor
 from ..pymilo_obj import Export, Import
 from .communicator import RESTClientCommunicator
 from ..transporters.general_data_structure_transporter import GeneralDataStructureTransporter
+
+
+class Mode(Enum):
+    """Mode is an enum class used to determine the fallback state of the PyMiloClient ."""
```

Comment:
`fallback state of the PyMiloClient.`
It's better if you can make it even simpler

Reply:
@sadrasabouri no tests are because of docstring (it will be fixed after merging the `docstrings` pr). it's ok, go ahead.